Dockerfile.amd64

```diff
@@ -11,10 +11,12 @@ RUN export GOCACHE=/tmp && \
     export GOARCH=${ARCH} && \
     export CGO_ENABLED=0 && \
     export GOOS=linux && \
+    export GIT_VERSION=`git rev-parse --short HEAD 2>/dev/null` && \
 	go build -o dist/images/rama -ldflags "-w -s" -v ./cmd/cni && \
-	go build -o dist/images/rama-daemon -ldflags "-w -s" -v ./cmd/daemon && \
-    go build -o dist/images/rama-manager -v ./cmd/manager && \
-    go build -o dist/images/rama-webhook -v ./cmd/webhook
+	go build -ldflags "-w -s -X \"main.gitVersion=`echo $GIT_VERSION`\" " -o dist/images/rama-daemon -v ./cmd/daemon && \
+    go build -ldflags "-X \"main.gitVersion=`git rev-parse --short HEAD 2>/dev/null`\" " -o dist/images/rama-manager -v ./cmd/manager && \
+    go build -o dist/images/rama-webhook -v ./cmd/webhook && \
+    echo $GIT_VERSION > ./VERSION
```

Comment:
VERSION seems puzzled to me, could we use another name?

Reply:
change to COMMIT_ID